ipyparallel/client/client.py

```diff
@@ -1326,9 +1326,16 @@ def become_dask(self, targets='all', port=0, nanny=False, scheduler_args=None, *
         worker_args.setdefault('ncores', 1)
         dview.apply_sync(util.become_dask_worker, **worker_args)
 
-        # Finally, return an Executor connected to the Scheduler
-        executor = distributed.Executor('{ip}:{port}'.format(**distributed_info))
-        return executor
+        # Finally, return a Client connected to the Scheduler
+        try:
+            distributed_Client = distributed.Client
+        except AttributeError:
+            # For distributed pre-1.18.1
+            distributed_Client = distributed.Executor
```

Comment:
I'm not sure if this fallback is strictly necessary as `Client` definitely exists pre-1.18.1. Though am unclear for how long it has been around.